### DISCUSSION TITLE
Flowcharts using networkx?

### DISCUSSION THREAD

**Discussion by 88arvin** [networkx/networkx#6786] [Q&A]
Can we create a flowchart with pandas dataframe? Something like this :

![Capture](https://github.com/networkx/networkx/assets/122347931/8d7bba16-b2cf-4252-9d03-69739880b231)

**Comment by dschult** [comment]
Our experience is that GraphViz is the best package for that kind of visualization.

We have a link to the python interface to GraphViz (pygraphviz). So you can create the network in networkx and use pygraphviz from nx to get an image file. It also might be better to have nx/pygraphviz to create a `.dot` file. and then you can adjust the dot file to get the details the way you want.

**Reply by 88arvin** [reply]
Could you please give me instructions on how to accomplish it?
